### PR TITLE
:tada: feat: upload middleware, create page SendPicture.jsx , and securise form upload image

### PR DESCRIPTION
**Pull request by Lambseb**

### CODE REVIEW

File: backend/src/controllers/artWorkControllers.js [hunk 1]
```diff
@@ -1,15 +1,31 @@
 const tables = require("../tables");
 
-const browse = async (req, res) => {
+// const browse = async (req, res) => {
```
Comment (Kino-ki) [inline backend/src/controllers/artWorkControllers.js:3]: dead code

File: frontend/index.html [hunk 2]
```diff
@@ -2,9 +2,13 @@
...
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/src/assets/favicon.svg" />
+    <link
+      rel="icon"
+      type="image/svg+xml"
+      href="./src/assets/Steve_redimentioned .png"
+    />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite App</title>
+    <title>MAPPING ART</title>
```
Comment (Kino-ki) [inline frontend/index.html:11]: nicee :))

File: frontend/src/pages/MapPage.jsx [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
-      <span className=" flex flex-row justify-center font-extrabold text-4xl md:text-5xl lg:text-6xl mb-4 text-primary mt-0">
+    <div className="lg:h-[88vh] ">
+      <span className=" flex flex-row justify-center  font-extrabold text-4xl md:text-5xl lg:text-6xl mb-4 text-primary mt-0">
         LA CARTE
       </span>
       <div className="">
         <LeafletMap dbartworks={artworks} />
+        <NavLink
+          to="/submitartwork"
+          className=" flex justify-center w-[20rem] mb-2 mx-auto cursor-pointer
```
Comment (Kino-ki) [inline frontend/src/pages/MapPage.jsx:15]: a button wrapped in a Link is best suited

**Review by Kino-ki — CHANGES_REQUESTED** [review]

**Review by Jubdc — APPROVED** [review]

File: frontend/src/pages/MapPage.jsx (hunk 3 above)
Comment (Lambseb) [reply frontend/src/pages/MapPage.jsx:15]: I can rework after PR thanks

**Pull request closed without merging**